datetime/src/main/java/com/fasterxml/jackson/datatype/jsr310/JavaTimeFeature.java

```diff
@@ -17,10 +17,20 @@ public enum JavaTimeFeature implements JacksonFeature
      * Default setting is enabled, for backwards-compatibility with
      * Jackson 2.15.
      */
-    NORMALIZE_DESERIALIZED_ZONE_ID(true)
-    ;
+    NORMALIZE_DESERIALIZED_ZONE_ID(true),
 
     /**
+     * Feature that controls whether numeric strings may be interpreted as numeric
+     * timestamps (enabled) or not (disabled), in addition to an explicitly
+     * defined pattern.
+     * <p>
+     * Note that when a pattern is not explicitly defined numeric strings are
+     * interpreted as a numeric timestamp.
+     */
+    ALWAYS_ALLOW_STRINGIFIED_TIMESTAMPS(false)
```

Comment:
To be consistent with `READ_DATE_TIMESTAMPS_AS_NANOSECONDS` I reckon `ALWAYS_READ_STRINGIFIED_DATE_TIMESTAMPS` would be more accurate.

Reply:
I'll add `DATE_` in there, good suggestion.

The reason I think "ALLOW" is important is because this is used in addition to -- but not instead of -- custom pattern.